deerlab/fitparamodel.py

```diff
@@ -26,10 +26,10 @@ def fitparamodel(V, model, par0=[],lb=[],ub=[], weights = 1,
     par0  : array_like
         Start values of the model parameters.
     
-    lb : array_like      
+    lb : array_like, optional      
         Lower bounds for the model parameters. If not specified, it is left unbounded.

```

Comment:
Is this unbounded-when-not-given behavior consistent across all fit functions?

Reply:
It is not. 
For `fitmultimodel` and `fitsignal` when not given the bounds are taken from the built-in models. Such an implementation is not possible for user-defined models as in `fitparamodel`.